pages/documentation/simulation/flowdecomposition/index.md

```diff
@@ -0,0 +1,183 @@
+- Internal flow: flow due to electricity exchange inside the network element's zone.
+- Loop flow: flow due to electricity exchange inside another zone.
+- PST flow: flow due to a shift commanded by the action of an active phase shifting transformer on the network.
+- HVDC flow: flow due to a shift commanded by the action of an active HVDC line on the network.
+
+> Current algorithm does not model HVDC flow yet.
+
+> This decomposition does not reflects the exact reality of how electric flows act on a real network, but it
+> is a useful approximation needed for some cross zonal coordination processes.
+
+## Algorithm description
+
+The flow decomposition algorithm is based on the DC approximation, in which the losses in the network branches
+are neglected, and that allows to rely on the superposition principle to assess which is the impact of any injection
+on any branch flow by simple sensitivity analysis.
+
+Below is the concrete description of the algorithm implemented in PowSyBl.
+
+![Flow decomposition algorithm chart](./img/flowDecompositionAlgorithmChart.svg)
+
+### Net positions computation
+
+Countries' net position computation is done using AC loadflow in the initial network, before any other alteration of the input.
+
+The net position of a country is calculated as the sum of :
+- The injection of all dangling lines connected in the country
+- The mean leaving flow of all AC  and HVDC line interconnections (losses are shared equally between both countries)
+
+### Losses compensation
+
+In order to mitigate the impact of DC approximation in the flow decomposition process, a dedicated step of losses compensation is implemented.
+
+Instead of using standard power flow compensation methodology, a standard full AC power flow is run on the input network
+that allows to calculate the losses on each network element.
+
+These losses are then compensated on the sending side of each network element.
+
+![Losses compensation on lines](./img/lossesCompensationOnLine.svg)
+
+A special treatment is done on tie lines, where instead of compensating the losses on the sending terminal, losses are
+compensated at both sides proportionally to the resistance of each half line.
+
+![Losses compensation on tie lines](./img/lossesCompensationOnTieLine.svg)
+
+### Nodal Injections partitioning
+
+In order to distinguish internal/loopflows and allocated flows, the nodal injections in each zone must de decomposed in two parts:
+- Nodal injections for allocated flows
+- Nodal injections for loop flows and internal flows
+
+This decomposition is based on GLSK (Generation and Load Shift Keys). It is an input of the process that provides,
+for each zone of the study a list of injections and associated factor to be used to scale the zone to a given net position.
+
+By default, the algorithm uses so-called "Country GSK", which is an automatic GLSK that scales on all generators
+proportionally to their target power set-point.
```

Comment:
setpoint

Reply:
My spell checker does not agree with you on that but I can change it if you want

Reply:
Indeed it is just to be coherent with the rest of the site, I am not so good in English you know that!

Reply:
For consistency with the rest of the documentation I put setpoint